pkg/notify/notify.go

```diff
@@ -8,20 +8,19 @@ import "github.com/ovh/utask"
 
 var (
 	senders = make(map[string]NotificationSender)
-	// Actions represents configuration of each notify actions
+	// actions represents configuration of each notify actions
 	actions utask.NotifyActions
 )
 
 // Payload is the holder of data to be sent as a notification
 type Payload interface {
-	Message() string
-	Fields() []string
+	Message() *Message
```

Comment:
I believe the Payload interface doesn't accomplish much anymore... why not pass around *Message directly?

Reply:
I done that because of the legacy architecture, fixed now.